ergo/platforms/almanis.py

```diff
@@ -0,0 +1,226 @@
+from abc import abstractmethod
+import json
+from typing import Dict, List, Sequence
+
+import requests
+
+from ergo.questions import BinaryQuestion, Question
+
+
+class Almanis:
+    """
+    The main class for interacting with Almanis
+    """
+
+    question_id_request_json = json.dumps(
+        {
+            "customerID": "44d7f0b3-7e6f-4867-b00a-6816492bb510",
+            "statuses": [{"Trading": {}}, {"ClosedForTrading": {}}],
+        }
+    )
+
+    BINARY_QUESTION_KEY = "BooleanQuestionOD"
+    CONTINUOUS_QUESTION_KEY = "MultipleChoiceQuestionOD"
+
+    def __init__(self):
+        self.api_url = "https://proddysruptlabsr53tobackendapi.dysruptlabs.com/api/sharedCode/api/IntegrationLayerApi"
+        self.s = requests.Session()
+        self.question_ids = []
+
+    def refresh_question_ids(self):
+        response = self.s.post(
+            f"{self.api_url}/getQuestionIDs", data=self.question_id_request_json
```

Comment:
can use the `json=` argument rather than `json.dumps` then `data=`

Reply:
Oh, yeah, this is much better, thanks!